### PR TITLE
Device change mixin

### PR DESCRIPTION
**Pull request by gcandal**

### CODE REVIEW

File: vue/mixins/device.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            if (width > tabletWidth) {
+                body.classList.add(DESKTOP_CLASS);
+            } else if (width > mobileWidth) {
+                body.classList.add(TABEL_CLASS);
+            } else {
+                body.classList.add(MOBILE_CLASS);
+            }
+        },
+        _getBody() {
+            if (this._body) return this._body;
```
Comment (joamag) [inline vue/mixins/device.js]: You know that `document.body` does the job right ;)?

File: vue/mixins/device.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        },
+        setMobileWidth(width, update = false) {
+            this.mobileWidth = width;
+            if (update) this.updateDeviceClass();
+        },
+        listenDeviceChange() {
+            window.addEventListener("resize", () => this.updateDeviceClass());
+        },
+        updateDeviceClass() {
+            const width = window.innerWidth;
```
Comment (joamag) [inline vue/mixins/device.js]: I've seen this code elsewhere... where was it?

Comment (gcandal) [reply vue/mixins/device.js]: https://github.com/ripe-tech/ripe-commons-pluginus/blob/master/js/util/device.js

File: vue/mixins/device.js (hunk 1 above)
Comment (gcandal) [reply vue/mixins/device.js]: (facepalm)

**Pull request merged**